tempto-core/src/main/java/io/prestosql/tempto/internal/convention/tabledefinitions/ConventionTableDefinitionsProvider.java

```diff
@@ -1,122 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package io.prestosql.tempto.internal.convention.tabledefinitions;
-
-import io.prestosql.tempto.fulfillment.table.TableDefinition;
-import io.prestosql.tempto.fulfillment.table.TableDefinitionsRepository;
-import io.prestosql.tempto.fulfillment.table.TableHandle;
-import io.prestosql.tempto.fulfillment.table.hive.HiveDataSource;
-import io.prestosql.tempto.fulfillment.table.hive.HiveTableDefinition;
-import io.prestosql.tempto.fulfillment.table.jdbc.RelationalDataSource;
-import io.prestosql.tempto.internal.convention.ConventionBasedTestFactory;
-import org.slf4j.Logger;
-
-import java.io.IOException;
-import java.nio.file.Path;
-import java.util.List;
-import java.util.Optional;
-import java.util.stream.StreamSupport;
-
-import static io.prestosql.tempto.fulfillment.table.TableHandle.tableHandle;
-import static io.prestosql.tempto.fulfillment.table.hive.HiveTableDefinition.hiveTableDefinition;
-import static io.prestosql.tempto.fulfillment.table.jdbc.RelationalTableDefinition.relationalTableDefinition;
-import static io.prestosql.tempto.internal.convention.ConventionTestsUtils.getConventionsTestsPath;
-import static java.nio.file.Files.exists;
-import static java.nio.file.Files.newDirectoryStream;
-import static java.util.Collections.emptyList;
-import static java.util.stream.Collectors.toList;
-import static org.slf4j.LoggerFactory.getLogger;
-
-public class ConventionTableDefinitionsProvider
```

Comment:
undo? Why to inline it?

Reply:
I wanted to have a better (and clear) control how `TABLE_DEFINITIONS_REPOSITORY` is used and where so I could control how we inject additional tables.